docs/trace.md

```diff
@@ -207,3 +275,14 @@ The meaning of the trace timestamps is:
 
   * Overhead: Additional time required for request handling not
     covered by Queue or Compute times.
+
+* Data Flow: The data flow of the first request. It contains the input and 
+  output tensors of each part of execution.
+
+  * Name: The name of model.
```

Comment:
Name and version should be at a higher level. What does a complete trace summary with TIMESTAMPS and TRACE look like?

Reply:
The "TIMESTAMPS" output is the same as before. The "TRACE" output is an extra display in the terminal. An example is shown in the lines from 204 to 238.
Each model entry has a "Name", "Version", "QUEUE_INPUT" and "BACKEND_OUTPUT". If we run an ensemble model, several model entries is shown hierarchically.